dts/bindings/sensor/st,lsm6dsv16x-common.yaml

```diff
@@ -59,10 +59,10 @@ properties:
     description: |
       Range in g. Default is power-up configuration.
 
-      - 0 # LSM6DSV16X_DT_FS_2G  (0.061 mg/LSB)
-      - 1 # LSM6DSV16X_DT_FS_4G  (0.122 mg/LSB)
-      - 2 # LSM6DSV16X_DT_FS_8G  (0.244 mg/LSB)
-      - 3 # LSM6DSV16X_DT_FS_16G (0.488 mg/LSB)
+      - 0 # LSM6DSV16X_DT_FS_2G  (0.061 mg/LSB) | LSM6DSV32X_DT_FS_4G  (0.122 mg/LSB)
+      - 1 # LSM6DSV16X_DT_FS_4G  (0.122 mg/LSB) | LSM6DSV32X_DT_FS_8G  (0.244 mg/LSB)
+      - 2 # LSM6DSV16X_DT_FS_8G  (0.244 mg/LSB) | LSM6DSV32X_DT_FS_16G (0.488 mg/LSB)
+      - 3 # LSM6DSV16X_DT_FS_16G (0.488 mg/LSB) | LSM6DSV32X_DT_FS_32G (0.976 mg/LSB)
```

Comment:
This is confusing, the description says `in g`, but the values are 0, 1, 2, 3. Also they change the meaning when moving to a different driver. Can we:
* Remove this attribute from 16x-common
* Rename 16x-common to xxx-common
* Recreate 16x-common with this attribute
* Create 32x-common with `accel-range` enum 4, 8, 16, 32
* Then use the 32x-common in the 32x drivers